test.sh

```diff
@@ -24,10 +24,7 @@ cd $HOME/go/src
 protoc -I=. --go_out=. github.com/googleinterns/step224-2020/config/proto/*.proto
 
 # Run all go tests
-# Find finds all of the files ending in _test.go and returns their relative
-# paths. Sed removes the the file part of that relative path.
-# Then, we remove uniques.
-go test $(find ./github.com/googleinterns/step224-2020 -type f -name '*_test.go' | sed -r 's|/[^/]+$||' -u)
+go test github.com/googleinterns/step224-2020/...
 
 # Run go fmt on all .go files to format them
 go fmt github.com/googleinterns/step224-2020/...
```

Comment:
I would swap the order of these two: first run gofmt and then run tests.

Reply:
That's a good point, thanks, I've reordered these.